frontend/src/components/Logout/logoutModal.css

```diff
@@ -0,0 +1,142 @@
+
+.modal-container{
+  min-height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+  padding-top: 1rem;
+  z-index: 500;
+  position: fixed;
+  top: 0;
+  background-color: rgba(0, 0, 0, 0.3);
+}
+
+.main-container {
+  width: 642px;
+  gap: 24px;
+  border-radius: 20px;
+  padding: 1rem;
+  color: white;
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card {
+  width: 309px;
+  height: 101px;
+  background: rgba(18, 7, 33, 1);
+  border: 1px solid rgba(54, 41, 78, 1);
+  border-radius: 900px;
+  padding-top: 4px;
+  padding-right: 24px;
+  padding-left: 24px;
+  text-align: center;
+}
+
+.card-header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: none;
+  color: white;
+}
+
+
+.card-value {
+  font-size: 24px;
+  font-weight: 600;
+  color: white;
+}
+
+.main-card {
+  background: rgba(18, 7, 33, 1);
+  border: 1px solid rgba(54, 41, 78, 1);
```

Comment:
use css variables if already exists, if not create new ones (check global.css and figma hex colors)